examples/24_gaussian_splats.py

```diff
@@ -67,6 +71,63 @@ def load_splat_file(splat_path: Path, center: bool = False) -> SplatFile:
     }
 
 
+def load_ply_file(ply_file_path: Path, center: bool = False) -> SplatFile:
+    plydata = PlyData.read(ply_file_path)
+    vert = plydata["vertex"]
+    sorted_indices = onp.argsort(
+        -onp.exp(vert["scale_0"] + vert["scale_1"] + vert["scale_2"])
+        / (1 + onp.exp(-vert["opacity"]))
+    )
+    colors = onp.zeros((len(sorted_indices), 3))
+    opacities = onp.zeros((len(sorted_indices), 1))
+    positions = onp.zeros((len(sorted_indices), 3))
+    wxyzs = onp.zeros((len(sorted_indices), 4))
+    scales = onp.zeros((len(sorted_indices), 3))
+    for idx in sorted_indices:
+        v = plydata["vertex"][idx]
+        position = onp.array([v["x"], v["y"], v["z"]], dtype=onp.float32)
+        scale = onp.exp(
+            onp.array([v["scale_0"], v["scale_1"], v["scale_2"]], dtype=onp.float32)
+        )
+
+        rot = onp.array(
+            [v["rot_0"], v["rot_1"], v["rot_2"], v["rot_3"]], dtype=onp.float32
+        )
+        SH_C0 = 0.28209479177387814
+        color = onp.array(
+            [
+                0.5 + SH_C0 * v["f_dc_0"],
+                0.5 + SH_C0 * v["f_dc_1"],
+                0.5 + SH_C0 * v["f_dc_2"],
+            ]
+        )
+        opacity = 1 / (1 + onp.exp(-v["opacity"]))
+        wxyz = ((rot / onp.linalg.norm(rot)) * 128 + 128).clip(0, 255).astype(
+            onp.uint8
+        ) / 255.0 * 2.0 - 1.0
```

Comment:
could you explain or add some comments for these multiplication? if `rot` isn't already unit norm, is `rot / onp.linalg.norm(rot)` not sufficient?

Reply:
changed to just rot / onp.linalg.norm(rot), honestly that was just for me looking through your code and combining it with antimatters convert.py without much thought